js/main.js

```diff
@@ -30,4 +30,60 @@ function handleSubmit(event) {
   data.entries.unshift($newEntry);
   $form.reset();
   $img.setAttribute('src', placeholder);
+  document.querySelector('.entries').className = 'entries';
+  document.querySelector('.entry-form').className = 'entry-form hidden';
+  document.location.reload();
+
+}
+
+function entryTree(entry) {
+  var notesText = document.createElement('p');
+  notesText.className = 'bottom-padding';
+  notesText.textContent = entry.notes;
+  var entryHeading = document.createElement('h1');
+  entryHeading.className = 'top-padding-one bottom-padding';
+  entryHeading.textContent = entry.title;
+
+  var entryImg = document.createElement('img');
+  entryImg.setAttribute('src', entry.photoUrl);
+
+  var halfText = document.createElement('div');
+  halfText.className = 'column-half bottom-padding';
+  halfText.appendChild(entryHeading);
+  halfText.appendChild(notesText);
+
+  var halfImg = document.createElement('div');
+  halfImg.className = 'column-half bottom-padding';
+  halfImg.appendChild(entryImg);
+
+  var block = document.createElement('div');
+  block.className = 'row entry';
+  block.appendChild(halfImg);
+  block.appendChild(halfText);
+  return block;
+}
+window.addEventListener('DOMContentLoaded', handleEntry);
+function handleEntry(event) {
+  var $ul = document.querySelector('ul');
+  for (var i = 0; i < data.entries.length; i++) {
+    $ul.appendChild(entryTree(data.entries[i]));
+  }
+}
+
+if (data.entries.length === 0) {
+  document.querySelector('.no-entries').className = 'no-entries';
+
+} else {
+  document.querySelector('.no-entries').className = 'no-entries hidden';
+}
+
+function handleClick(event) {
+  if (event.target.matches('.entries-tab')) {
+    document.querySelector('.entries').className = 'entries';
+  } else if (event.target.matches('.new-button')) {
+    document.querySelector('.entry-form').className = 'entry-form';
+    document.querySelector('.entries').className = 'entries hidden';
+  }
 }
```

Comment:
I would recommend creating a single function named something like `switchView` that receives an argument of the name of the view to switch to, and the function should show that view and hide all other views. That way, you won't need to make a new function for each new view created, or a new conditional block in this function.